### DISCUSSION TITLE
Specifying capture and monitoring features

### DISCUSSION THREAD

**Discussion by KingAkeem** [PSNAppz/ShadowGuard#4] [General]
The responsibilities of the capture and monitor features need to be clarified before we can work on fully implementing them.

- What is the capture feature? 
- What is the monitor feature?

Once those questions are answered, we can begin to work on executing either feature.

**Comment by PSNAppz** [comment]
Well, I think I should've named it better. `Capture` can be removed entirely. It's very similar to monitor in terms of feature. We are actually capturing the request before monitoring so, that doesn't make sense.
